app/scripts/app.js

```diff
@@ -17,7 +23,8 @@ blocChat.config(['$stateProvider', '$locationProvider', function($stateProvider,
 
 blocChat.factory("chatRooms", [
     "$firebaseArray",
-    function($firebaseArray) {
+    "CurrentRoom",
+    function($firebaseArray, CurrentRoom) {
         var roomsUrl = 'https://stewart-bloc-chat.firebaseio.com/rooms';
         var roomsRef = new Firebase(roomsUrl);
         var roomsObject = $firebaseArray(roomsRef);
```

Comment:
what's weird, is that if I modify this line to be: 

```var roomsObject = $firebaseArray(roomsRef.orderByChild('name').equalTo('villains'));```

It works. It only displays the villains room in the Rooms list.